lib/elixir/lib/supervisor.ex

```diff
@@ -610,7 +610,7 @@ defmodule Supervisor do
     * `:max_restarts` - the maximum number of restarts allowed in
       a time frame. Defaults to `3`.
 
-    * `:max_seconds` - the time frame in which `:max_restarts` applies.
+    * `:max_seconds` - the time frame *in seconds* in which `:max_restarts` applies.
```

Comment:
This should be clear from the name, I don't think we need to point it out with `*`.

Reply:
We can remove the emphasis